openhands/runtime/plugins/agent_skills/file_reader/file_readers.py

```diff
@@ -103,7 +103,9 @@ def _base64_video(file_path: str, frame_interval: int = 10) -> list[str]:
     return base64_frames
 
 
-def _prepare_image_messages(task: str, base64_image: str):
+def _prepare_image_messages(
+    task: str, base64_image: str
+) -> list[dict[str, list[dict[str, str | dict[str, str]]] | str]]:
```

Comment:
```suggestion
) -> list[dict[str, Any]]:
```

The type here is correct but maybe overly so. We often use `dict[str, Any]` for messages in this form. If that works downstream `list[dict[str, Any]]` might be a simpler return value here.

Reply:
I'd slightly prefer to stick with the overly verbose type here because an overly verbose type shows that we're doing something a bit too complicated and may want to simplify in the future (such as define an actual datatype).